src/db/migrations/29-add-holly-plus.ts

```diff
@@ -23,6 +23,7 @@ const HOLLY_PLUS: NftFactory = {
     overrides: {
       artist: {
         name: 'hollyplus',
+        artistId: 'hollyplus',
```

Comment:
why is this needed?

Reply:
It was previously falling back to platform id, which was `hollyplus`.

Now that we've removed that bad default, I've added this to be explicit about what was here before.

Reply:
also should we use ethereumArtistId(address) if we do need this? or maybe make ethereumArtistId(nftFactoryAddress) a default that everything uses

Reply:
> It was previously falling back to platform id, which was `hollyplus`.
> 
> Now that we've removed that bad default, I've added this to be explicit about what was here before.

ah ok - well since hollyplus hasn't actually been released/deployed yet, maybe the better default of ethereumArtistId(nftFactoryAddress) is fine for it?